CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,289 @@
+============================================================
+Tor Project Code of Conduct
+============================================================
```

Comment:
We should probably reformat this to be markdown or maybe just link to the canonical version of this hosted on some torproject website.

Reply:
Changing it to markdown.